src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.test.ts

```diff
@@ -560,6 +560,52 @@ describe('WorkspaceSavedObjectsClientWrapper', () => {
           },
         });
       });
+      it('should call client.find with ACLSearchParams if no workspaces is provided', async () => {
+        const { wrapper, clientMock } = generateWorkspaceSavedObjectsClientWrapper();
+        // no workspaces
+        await wrapper.find({
+          type: 'dashboards',
+        });
+        expect(clientMock.find).toHaveBeenCalledWith({
```

Comment:
Since `clientMock.find` will be called multi times, prefer using `toHaveBeenLastCalledWith`

Reply:
good catch.